src/Domain/Sale/Application/CashierService.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This file is part of the php-ddd project.
+ *
+ * (c) Yannick Voyer <star.yvoyer@gmail.com> (http://github.com/yvoyer)
+ */
+
+namespace Example\Domain\Sale\Application;
+
+use Example\Domain\Administration\DomainModel\Event\CandidateWasHired;
+use Example\Domain\Common\Application\EventPublisher;
+use Example\Domain\Common\DomainModel\JobTitle;
+use Example\Domain\Sale\DomainModel\Cashier;
+use Example\Domain\Sale\DomainModel\EmployeeRepository;
+use Example\Domain\Sale\DomainModel\Identity\EmployeeId;
+
+final class CashierService
```

Comment:
Could be renamed to a single responsabilty name like `CashierCreator` or anything else.

@webdevilopers any idea about better names in this pr?